### DISCUSSION TITLE
2021 datafuse labs T-shirt is coming!

### DISCUSSION THREAD

**Discussion by drmingdrmer** [databendlabs/databend#3596] [General]
This is a small gift for contributors who have been working on databend (a datafuselabs project).

Like it or not like it?
We are listening! https://github.com/datafuselabs/datafuse-shop/pull/1

![2021-tshirt-smaller-font](https://user-images.githubusercontent.com/44069/123891918-38693580-d98c-11eb-8b07-0bf2cbc1486a.png)

- `data`: with cloud as background;
  Using a classic font to represent enterprise data on cloud.

- `fuse` and the trailing underline cursor:
  Using traditional font and color that widely used on a computer terminal.
  The cursor is a symbol that we've been actively hacking and hacking on this product.

**Comment by bohutang** [comment]
Hello Datafuse Contributors, 

Our 2021 T-shirt will be released soon, please email us with your t-shirt **sizing** and mailing **address&phone&github ID** to **hi@datafuselabs.com**

Contributors:
@leiysky
@tlightsky
@TennyZhuang
@b41sh
@smallfish
@JmPotato
@taiyang-li
@zhaox1n
@tisonkun
@TCeason
@wangzhen11aaa 
@loadlj 
@gauss1314
@SGZW
@yjhmelody
@xuyifangreeneyes
@yjhmelody
@ives9638
@cppcoffee
@junli1026 
@jackwener
@koushiro
@jyz0309
@mapleFU
@ygf11
@GrapeBaBa
@tangwz
@keliwang
@Veeupup
@yufan022
@Junnplus
@sunisdown
@yuuch 
@lroolle 
@sunli829
@kyon0304
@Chasen-Zhang
@pinylin
@miamia0
@mshauneu
@xxchan 
@QuantumBear
@unconsolable
@radmirnovii
@eastfisher
@lianghanzhen
@weakish

If anyone is missing, please leave a comment and let me know. Thanks!

Shirt Sizing Charts:
![datafuse-t-shirt-size](https://user-images.githubusercontent.com/172204/124140953-89c41280-dabb-11eb-85fd-2cf914304ab5.jpg)

**Comment by zhang2014** [comment]
> If anyone is missing, please leave a comment let me know, thanks!

@BohuTANG 
And a new contributor, @wangzhen11aaa #947

**Comment by bohutang** [comment]
If you want a 2021 datafuse t-shirt, please join our `system.contributors` system table:

```
datafuse :) SELECT *　FROM system.contributors;

SELECT *
FROM system.contributors

Query id: b10317f8-9401-401a-be17-3a7f566df920

┌─name────────────────────┐
│ artorias1024            │
│ b41sh                   │
│ BohuTANG                │
│ Chojan Shang            │
│ dantengsky              │
│ DatafuseBot             │
│ dependabot[bot]         │
│ dependabot-preview[bot] │
│ drdr xp                 │
│ Eason                   │
│ Frank.Wang              │
│ hulunbier               │
│ JmPotato                │
│ jyizheng                │
│ leiysky                 │
│ Qing Wang               │
│ smallfish               │
│ sundy-li                │
│ sundyli                 │
│ taiyang-li              │
│ TennyZhuang             │
│ tison                   │
│ TLightSky               │
│ wangzhen05              │
│ Winter Zhang            │
│ wubx                    │
│ yizheng                 │
│ Yizheng Jiao            │
│ zhang2014               │
│ zhaox1n                 │
│ zhihanz                 │
│ ZhiHanZ                 │
│ Zhihan Zhang            │
│ 张炎泼                  │
│ 陈小玉                  │
└─────────────────────────┘

35 rows in set. Elapsed: 0.003 sec. 
```

How to JOIN:
https://datafuse.rs/development/contributing/

**Comment by wangzhen11aaa** [comment]
I want to change the name to frank-wang. :)

**Comment by sundy-li** [comment]
> I want to change the name to frank-wang. :)

It's auto-generated by scripts using your git commit user, so I'm sorry we can't make it special.

**Comment by zhaox1n** [comment]
Prefer use the email address instead of git commit user. It seems that duplicate names in this list

**Comment by bohutang** [comment]
@wangzhen11aaa @zhaox1n 
As sundy said, they are generated from:
https://github.com/datafuselabs/datafuse/blob/master/common/building/src/lib.rs#L36-L45
ClickHouse style

**Comment by bohutang** [comment]
@loadlj +1
Lucky star contributor :+1: 

![image](https://user-images.githubusercontent.com/172204/126626216-c259b80e-fc98-4915-99e2-02f1c60235bf.png)
